### PR TITLE
Adding package-specific tags in DESCRIPTION files of the model packages

### PR DESCRIPTION
**Pull request by Parthmadan-1**
This is a WIP PR, kindly provide your suggestions.

### CODE REVIEW

**Comment by mdietze** [comment]
Could you provide a more informative PR title and description?

File: models/sipnet/DESCRIPTION [hunk 1]
```diff
@@ -33,3 +33,5 @@ LazyData: FALSE
 Encoding: UTF-8
 Require: 
 RoxygenNote: 7.1.2
+X-schema.org-keywords: R, Predictive Ecosystem Analyzer, Carbon, Water, vegetated ecosystem , atmosphere, high speed model, Bruteforce Suitable, SIPNET
```
Comment (Aariq) [inline models/sipnet/DESCRIPTION:36]: What is "Bruteforce Suitable"?

Comment (Parthmadan-1) [reply models/sipnet/DESCRIPTION:36]: I read this online "SIPNET is a fast model (∼ 5.5 sec per MCMC iteration in PEcAn including model
execution, and writing and reading model outputs), which makes it suitable for application of bruteforce methods."

**Comment by Aariq** [comment]
The purpose of these keywords is so that someone searching for them on [r-universe](https://r-universe.dev/search/) would find our packages.  So some of these additions are probably not relevant.  Think about a domain scientist looking for a suitable R package.  What would they type in to find something like PEcAn.SIPNET?

File: models/ed/DESCRIPTION [hunk 2]
```diff
@@ -66,3 +66,5 @@ Encoding: UTF-8
 RoxygenNote: 7.1.2
 Roxygen: list(markdown = TRUE)
 Config/testthat/edition: 2
+X-schema.org-keywords: R, Predictive Ecosystem Analyzer,Ecosystem Demography, terrestrial biosphere, plant community dynamics, biogeochemical,soil fluxes, Carbon, Water, Nitrogen, SAS approximation, ED2
```
Comment (mdietze) [inline models/ed/DESCRIPTION:69]:
```suggestion
X-schema.org-keywords: Predictive Ecosystem Analyzer, Ecosystem Demography model, terrestrial biosphere model, vegetation demographic model, plant community dynamics, biogeochemistry, Carbon cycle, size- and age-structured approximation, ED2 
```

File: models/sipnet/DESCRIPTION (hunk 1 above)
Comment (mdietze) [inline models/sipnet/DESCRIPTION:36]:
```suggestion
X-schema.org-keywords: Predictive Ecosystem Analyzer, Carbon cycle, ecosystem model, SIPNET
```

File: modules/data.atmosphere/DESCRIPTION [hunk 3]
```diff
@@ -77,3 +77,5 @@ LazyLoad: yes
 LazyData: FALSE
 Encoding: UTF-8
 RoxygenNote: 7.1.2
+X-schema.org-keywords: R, Predictive Ecosystem Analyzer, data.atmosphere, meteorological products conversion, climate driver data
```
Comment (mdietze) [inline modules/data.atmosphere/DESCRIPTION:80]:
```suggestion
X-schema.org-keywords: Predictive Ecosystem Analyzer, meteorological data product conversion, climate driver data, downscaling, gap-filling
```

File: modules/meta.analysis/DESCRIPTION [hunk 4]
```diff
@@ -48,3 +48,5 @@ LazyData: FALSE
 Encoding: UTF-8
 RoxygenNote: 7.1.2
 Roxygen: list(markdown = TRUE)
+X-schema.org-keywords: R, Predictive Ecosystem Analyzer,meta analysis, plant traits, Bayesian
```
Comment (mdietze) [inline modules/meta.analysis/DESCRIPTION:51]:
```suggestion
X-schema.org-keywords: Predictive Ecosystem Analyzer, meta-analysis, plant traits, Bayesian
```

**Comment by infotroph** [comment]
Thanks for your work on this, but closing because the PEcAn team doesn't have bandwidth to complete it. New PRs on this topic are welcome from anyone, but be aware that the bulk of the work here is picking the right tags, which requires some domain expertise.

**Pull request closed without merging**